File: ContainDuplicate.py
class Solution(object):
    def containsDuplicate(nums):
        mapa={}

        for num in nums:
            if mapa.get(num):
                return True
            mapa[num] = str(num)
        
        return False
    
print(Solution.containsDuplicate(nums=[1,1,1,3,3,4,3,2,4,2]))

File: javascript/Containduplicate.js
var containsDuplicate = function(nums) {
    const map={}
    for (let i=0;i<nums.length;i++){
        num=nums[i]
        if(map[num]){
            return true;
        }
        map[num]=num.toString()
    }
    return false;
    
    
};

console.log(containsDuplicate([1,2,3,4,4]))

